code_fragments/DMIS_observer_match.sas

```diff
@@ -3,15 +3,15 @@
 libname test '~gardini/SAS/DMIS_Observer/only_2019';
 
 proc sql;
-	connect to oracle (user = gardini password = "XXXXXXXXXXXX" path = sole);
+	connect to oracle (user = gardini password = "XXXXXXXXXXXX" path = nova);
```

Comment:
nova?

Reply:
nova is the connection string that currently works on the SAS container.